test/registry.test.ts

```diff
@@ -12,11 +13,9 @@ type Pubkey = mcl.mclG2;
 
 function pubkeyToLeaf(p: Pubkey) {
     const uncompressed = mcl.g2ToHex(p);
-    const leaf = web3.utils.soliditySha3(
-        { t: "uint256", v: uncompressed[0] },
-        { t: "uint256", v: uncompressed[1] },
-        { t: "uint256", v: uncompressed[2] },
-        { t: "uint256", v: uncompressed[3] }
+    const leaf = ethers.utils.solidityKeccak256(
+        ["uint256", "uint256", "uint256", "uint256"],
+        uncompressed
```

Comment:
The typescript complains `soliditySha3` might return null type, so I changed those web3.utils.soliditySha3 to ethers.utils.solidityKeccak256.